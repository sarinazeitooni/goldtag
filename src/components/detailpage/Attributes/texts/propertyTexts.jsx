const propertyTexts = {
    properties : ' ویژگی ها'
};
export default propertyTexts;