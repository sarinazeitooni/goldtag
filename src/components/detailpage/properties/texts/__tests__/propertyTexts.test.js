import React from 'react';
import { shallow } from 'enzyme';
import propertyTexts from "../propertyTexts";
describe("propertyTexts text" , ()=>{
    it("it should render",()=>{
        const container = shallow(<propertyTexts/>)
        expect(container).toMatchSnapshot();
    });
});