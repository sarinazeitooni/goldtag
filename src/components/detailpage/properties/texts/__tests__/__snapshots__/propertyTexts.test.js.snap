// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`propertyTexts text it should render 1`] = `ShallowWrapper {}`;
